UFO-level-1/static/js/app.js

```diff
@@ -25,6 +25,7 @@ function buildTable(date_query,city_query) {
     var durationMinutes = [];
     var comments = [];
     
+    // comment right here!
```

Comment:
Might be better to implement a forEach loop that wraps a function which builds each table row: 
```
function buildTableRow(tableObj, datum) { 
    tableRow = tableObj.append("tr");
    tableRow.append("td").text(datum.dateTime);
    tableRow.append("td").text(datum.city);
....
}
```
```
data.forEach(datum => {
     buildTableRow(tableObj, datum);
});

```